client/static/index.css

```diff
@@ -97,6 +97,14 @@
   --candidate-in-progress: #2560ab;
   --candidate-approved: #309d08;
 
+  --success-green: #155724;
+  --success-green-bg: #d4edda;
+  --success-green-border-color: #c3e6cb;
+
+  --error-red: #721c24;
+  --error-red-bg: #f8d7da;
+  --error-red-border-color: #f5c6cb;
+
```

Comment:
These feels repetitive. I wonder if existing variables could be the baseline for green and red? I just want to promote a limited palette.